corehq/apps/reports/filters/users.py

```diff
@@ -453,6 +453,30 @@ def for_reporting_group(cls, group_id):
         }
 
 
+class ChangedByUserFilter(ExpandedMobileWorkerFilter):
+    slug = "changed_by_user"
+    label = ugettext_lazy("By User(s)")
+
+    def get_default_selections(self):
+        return [('t__6', _("[Web Users]"))]
+
+
+class ChangeActionFilter(BaseMultipleOptionFilter):
+    ALL = '0'
```

Comment:
All other values for options are integers ('1' / '2' / '3'), so kept it in line with that.
The option values are a string, to avoid conversion when checking for params which are always string.